fastexcel-reader/src/main/java/org/dhatim/fastexcel/reader/RowSpliterator.java

```diff
@@ -172,8 +169,7 @@ private Cell parseInlineStr(CellAddress addr) throws XMLStreamException {
         while (r.goTo(() -> r.isStartElement("is") || r.isEndElement("c") || r.isStartElement("f"))) {
             if ("is".equals(r.getLocalName())) {
                 rawValue = r.getValueUntilEndElement("is");
-                XSSFRichTextString rtss = new XSSFRichTextString(rawValue);
-                value = rtss.toString();
+                value = rawValue;
```

Comment:
Can inline string actually contain rich text? Current xlsx test files do not have such occurrences